src/goto-symex/goto_symex.h

```diff
@@ -76,7 +76,8 @@ class goto_symext
       path_storage(path_storage),
       path_segment_vccs(0),
       _total_vccs(std::numeric_limits<unsigned>::max()),
-      _remaining_vccs(std::numeric_limits<unsigned>::max())
+      _remaining_vccs(std::numeric_limits<unsigned>::max()),
+      global_validation_mode(validation_modet::IGNORE)
```

Comment:
Wouldn't it be possible to look at `options` instead and store that in a Boolean, as is done for several other entries in `options` above?

Reply:
Well I can use `options` in `goto_symext` but not in `statet` (unless I pass it there somehow).